pages/dashboard.vue

```diff
@@ -152,26 +153,35 @@ export default Vue.extend ({
       const currentProducts = products.filter(
         product => product.subscriptionStatus === ProductStatus.ACTIVE
       )
-      const isMhrPpr = hasMhrAndPprProducts(currentProducts)
 
       // only show products with no placeholder
       for (let i = 0; i < currentProducts.length; i++) {
-        if (currentProducts[i].code === ProductCode.BUSINESS_SEARCH) {
-          if (!getFeatureFlag('bcregistry-ui-bus-search-enabled')) continue
+        const enabledFF = `bcregistry-ui-${currentProducts[i].code}-enabled`
+        // ensure has a default (otherwise it might not have an enabled flag set in LD)
+        if (hasDefaultValue(enabledFF) && !getFeatureFlag(enabledFF)) {
```

Comment:
now future products we don't want visible just need the flag and the default (no specific code logic)